### DISCUSSION TITLE
Web Interface Language?

### DISCUSSION THREAD

**Discussion by Radoom** [TomBursch/kitchenowl#321] [Q&A]
Hi,
just came across this project, give it a try.  

Docker up and running, user created, logged in, so far all good and easy. No household set up yet, first wanted to switch the web UI to my native language - but can't find anything for that in settings in the web UI, nor in the advanced options documentation? Is the web interface and server side always English?

Edit: Language can be set for a household, giving items and categories (presets) in the selected language. But for the web UI itself, e.g. navigation items like Shopping List, Recipies, Settings... all english only. Not good for the WAF ;-)
 /Edit

(sorry if I don't see it ... maybe too late over here)

**Comment by TomBursch** [comment]
Sorry for the late reply, the interface language should follow your browser/device settings automatically if the language is available on [Weblate](https://hosted.weblate.org/engage/kitchenowl/). You are welcome to upvote the related issue, https://github.com/TomBursch/kitchenowl/issues/235.

**Reply by Radoom** [reply]
Merry Xmas and thanks for your response!!

Indeed, it does as you describe now. Tend to have my systems set to EN everywhere (so that when following guides/solutions etc around the web I don't have to double think what should be what in my case) though the native language for the rest of the family is different. As commented in #235, I'd also agree setting it per site / application in the respective settings would be more straight forward and more flexible. Using system's stettings as default maybe unless the user settings inside KitchenOwl (web client or app on mobile) are set. That way, people who are fine with their system settings driven language have to do nothing at all (don't even notice most likely), but others have the flexibility. That's all cosmetics though! Happy holidays :)
